monocle/db.py

```diff
@@ -304,12 +304,31 @@ class Fort(Base):
         order_by='FortSighting.last_modified'
     )
 
+class FortDetail(Base):
+    __tablename__ = 'fort_detail'
 
+    id = Column(Integer, primary_key=True)
+    external_id = Column(String(35), ForeignKey('forts.external_id'))
+    player_name = Column(String(100), index=True)
+    player_level = Column(Integer)
```

Comment:
maybe it would be good to have a extra player table and then only link the id here